SourceX/storm/storm.cpp

```diff
@@ -228,7 +246,7 @@ BOOL SBmpLoadImage(const char *pszFileName, SDL_Color *pPalette, BYTE *pBuffer,
 	}
 
 	if (pPalette && pcxhdr.BitsPerPixel == 8) {
-		SFileSetFilePointer(hFile, -768, 0, 1);
+		SFileSetFilePointer(hFile, -768, NULL, DVL_FILE_CURRENT);

```

Comment:
is DVL_FILE_CURRENT the proper def to use when talking to storm?

Reply:
We don't have any other defs for this as I understand

Reply:
I thought you defined these specifically to talk to Storm?